packages/dashboard/src/pages/Wallets/components/Transfer/TransferERC721.tsx

```diff
@@ -97,11 +112,23 @@ export const TransferERC721 = memo(() => {
 
     const allFormFields = watch()
 
+    //#region resolve ENS domain
+    const {
+        value: registeredAddress = '',
```

Comment:
Check if it is a valid address in the `useLookupAddress` hook. If so, return it directly.

Reply:
is already valid, the default value is because `useAsync`